kie-wb-common-react/boxed-expression-editor/src/components/Table/Table.css

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.table-component table {
+  font-style: normal;
+}
+
+.table-component table th {
+  position: relative;
+  border-style: ridge;
+  /** Due to a glitch when resizing column, disabling user selection for header's cells */
+  user-select: none;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+}
+
+.table-component table th .header-cell {
+  height: 3em;
+}
+
+.table-component table th .data-type {
+  font-style: italic;
+  font-size: smaller;
+}
+
+.table-component table th:not(:first-child) {
+  cursor: pointer;
+  text-align: center;
+  background-color: #DEF3FF;
+}
+
+.table-component table tbody td {
+  border-style: ridge;
+}
+
+.table-component table tbody td.row-count-cell {
+  display: inline-block;
+  box-sizing: border-box;
+  width: 60px;
+}
+
+.table-component table .pf-c-drawer__splitter {
+  background-color: #DEF3FF;
+}
+
+.table-component table .pf-c-drawer {
+  --pf-c-drawer__splitter--after--BorderLeftWidth: var(
+    --pf-global--BorderWidth--sm
+  );
+  position: absolute;
+  min-width: 0.625rem;
+  right: 0;
+  top: 0;
+  bottom: 0;
+}
+
+.table-component table tbody td textarea {
+  padding: 0.125em;
+  margin: 0;
+  border: 0;
+  width: 100%;
+}
```

Comment:
```suggestion
.table-component table tbody td textarea {
  padding: 0.125em;
  margin: 0;
  border: 0;
  width: 100%;
  height: 100%;
  resize: none;
}
```

These two CSS changes in the `Table.css` changes the cell from this:
![2021-01-14 10 18 38](https://user-images.githubusercontent.com/1079279/104570549-606d1280-5652-11eb-8db5-c33c5b7fe1c9.gif)

to this:
![2021-01-14 10 54 26](https://user-images.githubusercontent.com/1079279/104574964-f0ad5680-5656-11eb-80b7-e09f4b350465.gif)

Which is a bit closer from what we have in the old editor:
<img width="645" alt="image" src="https://user-images.githubusercontent.com/1079279/104575031-0458bd00-5657-11eb-84f4-e2951a976bbc.png">


Please, let me know wdyt :-)